src/renderer/atlas/AtlasEngine.h

```diff
@@ -530,6 +548,8 @@ namespace Microsoft::Console::Render
                 return _data.data();
             }
 
+            CachedGlyphLayout cachedLayout;

```

Comment:
`AtlasValue` isn't supposed to be POD, right

Reply:
Yes only `AtlasValueData` needs to be POD.